model/user.go

```diff
@@ -244,8 +244,8 @@ func (u *User) ToJson() string {
 }
 
 // Generate a valid strong etag so the browser can cache the results
-func (u *User) Etag() string {
-	return Etag(u.Id, u.UpdateAt)
```

Comment:
Instead of passing in those arguments, would it make sense to just pass in more of the user data into the Etag function? 0/5 on that, just an idea